$NetBSD$

Increase limits for data segment size and number of open files per process
to hard limits of system.

--- scripts/news.daily.in.orig	2019-02-07 15:02:22.000000000 +0000
+++ scripts/news.daily.in
@@ -276,6 +276,10 @@ fi
 
 cd ${PATHETC}
 
+## Unlimit
+ulimit -d `ulimit -H -d`
+ulimit -n `ulimit -H -n`
+
 ##  Show the status of the news system.
 ${DOSTAT} && {
     ${INNSTAT}
